spec/models/external_registries/stop_heling_bike_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+module ExternalRegistries
+  RSpec.describe ExternalRegistries::StopHelingBike, type: :model do
+    let(:api_response) do
+      {}.tap do |h|
```

Comment:
Why tap here rather than just defining a hash?

Reply:
`tap` is a sensible default because it's often useful to have each insertion happen in its own expression, in case there's logic surrounding the insertion. With a hash literal, it gets very cluttered and you end up cramming ternaries into the literal, for example. Updated in afe4daea

Reply:
Ok, I'm fine defaulting to tap